eng/verify-nupkgs.ps1

```diff
@@ -19,13 +19,13 @@ function Verify-Nuget-Packages {
     $expectedNumOfFiles = @{
         "Microsoft.CodeCoverage"                      = 59;
         "Microsoft.NET.Test.Sdk"                      = 15;
-        "Microsoft.TestPlatform"                      = 609;
+        "Microsoft.TestPlatform"                      = 614;
         "Microsoft.TestPlatform.Build"                = 20;
-        "Microsoft.TestPlatform.CLI"                  = 471;
+        "Microsoft.TestPlatform.CLI"                  = 473;
         "Microsoft.TestPlatform.Extensions.TrxLogger" = 34;
         "Microsoft.TestPlatform.ObjectModel"          = 92;
-        "Microsoft.TestPlatform.AdapterUtilities"     = 75;
-        "Microsoft.TestPlatform.Portable"             = 592;
+        "Microsoft.TestPlatform.AdapterUtilities"     = 61;
+        "Microsoft.TestPlatform.Portable"             = 604;
```

Comment:
differences:

```
Folders are the same.


Microsoft.TestPlatform is not the same
    Missing in new:
       Microsoft.TestPlatform\tools\<netfx>\Common7\IDE\Extensions\TestPlatform\System.Runtime.InteropServices.RuntimeInformation.dll
    Added in new:
       Microsoft.TestPlatform\tools\<netfx>\Common7\IDE\Extensions\TestPlatform\testhost.net481.arm64.exe
       Microsoft.TestPlatform\tools\<netfx>\Common7\IDE\Extensions\TestPlatform\testhost.net481.arm64.exe.config
       Microsoft.TestPlatform\tools\<netfx>\Common7\IDE\Extensions\TestPlatform\testhost.net481.exe
       Microsoft.TestPlatform\tools\<netfx>\Common7\IDE\Extensions\TestPlatform\testhost.net481.exe.config
       Microsoft.TestPlatform\tools\<netfx>\Common7\IDE\Extensions\TestPlatform\testhost.net481.x86.exe
       Microsoft.TestPlatform\tools\<netfx>\Common7\IDE\Extensions\TestPlatform\testhost.net481.x86.exe.config
    Old: 608 (609 in verify-nupkg, because of checksum file)
    New: 613 (614 in verify-nupkg, because of checksum file)
    You are skipping tfms.
```

The removed library moved to runtime, and we are adding 6 files for net481 testhost  609-1+6=614

```
Microsoft.TestPlatform.AdapterUtilities is not the same
    Missing in new:
       Microsoft.TestPlatform.AdapterUtilities\lib\<netcore>\Microsoft.TestPlatform.AdapterUtilities.dll
       Microsoft.TestPlatform.AdapterUtilities\lib\<netcore>\cs\Microsoft.TestPlatform.AdapterUtilities.resources.dll
       Microsoft.TestPlatform.AdapterUtilities\lib\<netcore>\de\Microsoft.TestPlatform.AdapterUtilities.resources.dll
       Microsoft.TestPlatform.AdapterUtilities\lib\<netcore>\es\Microsoft.TestPlatform.AdapterUtilities.resources.dll
       Microsoft.TestPlatform.AdapterUtilities\lib\<netcore>\fr\Microsoft.TestPlatform.AdapterUtilities.resources.dll
       Microsoft.TestPlatform.AdapterUtilities\lib\<netcore>\it\Microsoft.TestPlatform.AdapterUtilities.resources.dll
       Microsoft.TestPlatform.AdapterUtilities\lib\<netcore>\ja\Microsoft.TestPlatform.AdapterUtilities.resources.dll
       Microsoft.TestPlatform.AdapterUtilities\lib\<netcore>\ko\Microsoft.TestPlatform.AdapterUtilities.resources.dll
       Microsoft.TestPlatform.AdapterUtilities\lib\<netcore>\pl\Microsoft.TestPlatform.AdapterUtilities.resources.dll
       Microsoft.TestPlatform.AdapterUtilities\lib\<netcore>\pt-BR\Microsoft.TestPlatform.AdapterUtilities.resources.dll
       Microsoft.TestPlatform.AdapterUtilities\lib\<netcore>\ru\Microsoft.TestPlatform.AdapterUtilities.resources.dll
       Microsoft.TestPlatform.AdapterUtilities\lib\<netcore>\tr\Microsoft.TestPlatform.AdapterUtilities.resources.dll
       Microsoft.TestPlatform.AdapterUtilities\lib\<netcore>\zh-Hans\Microsoft.TestPlatform.AdapterUtilities.resources.dll
       Microsoft.TestPlatform.AdapterUtilities\lib\<netcore>\zh-Hant\Microsoft.TestPlatform.AdapterUtilities.resources.dll
    Added in new:

    Old: 74 (75 in verify-nupkg, because of checksum file)
    New: 60 (61 in verify-nupkg, because of checksum file)
    You are skipping tfms.
```

We removed 14 libraries for net6.0, because we don't target it anymore. 74-14=60

```
Microsoft.TestPlatform.CLI is not the same
    Missing in new:
       Microsoft.TestPlatform.CLI\contentFiles\any\<netcore>\TestHostNetFramework\datacollector.arm64.exe
       Microsoft.TestPlatform.CLI\contentFiles\any\<netcore>\TestHostNetFramework\datacollector.arm64.exe.config
       Microsoft.TestPlatform.CLI\contentFiles\any\<netcore>\TestHostNetFramework\datacollector.exe
       Microsoft.TestPlatform.CLI\contentFiles\any\<netcore>\TestHostNetFramework\datacollector.exe.config
    Added in new:
       Microsoft.TestPlatform.CLI\contentFiles\any\<netcore>\TestHostNetFramework\testhost.net481.arm64.exe
       Microsoft.TestPlatform.CLI\contentFiles\any\<netcore>\TestHostNetFramework\testhost.net481.arm64.exe.config
       Microsoft.TestPlatform.CLI\contentFiles\any\<netcore>\TestHostNetFramework\testhost.net481.exe
       Microsoft.TestPlatform.CLI\contentFiles\any\<netcore>\TestHostNetFramework\testhost.net481.exe.config
       Microsoft.TestPlatform.CLI\contentFiles\any\<netcore>\TestHostNetFramework\testhost.net481.x86.exe
       Microsoft.TestPlatform.CLI\contentFiles\any\<netcore>\TestHostNetFramework\testhost.net481.x86.exe.config
    Old: 470 (471 in verify-nupkg, because of checksum file)
    New: 472 (473 in verify-nupkg, because of checksum file)
    You are skipping tfms.
```

We removed 4 libraries that should not have been shipped, datacollector will be run from the main folder via datacollector.dll, and we add 6 files for testhost 481. So 2 new files added in total.

```
Microsoft.TestPlatform.Portable is not the same
    Missing in new:

    Added in new:
       Microsoft.TestPlatform.Portable\tools\<netfx>\testhost.net481.arm64.exe
       Microsoft.TestPlatform.Portable\tools\<netfx>\testhost.net481.arm64.exe.config
       Microsoft.TestPlatform.Portable\tools\<netfx>\testhost.net481.exe
       Microsoft.TestPlatform.Portable\tools\<netfx>\testhost.net481.exe.config
       Microsoft.TestPlatform.Portable\tools\<netfx>\testhost.net481.x86.exe
       Microsoft.TestPlatform.Portable\tools\<netfx>\testhost.net481.x86.exe.config
       Microsoft.TestPlatform.Portable\tools\<netcore>\TestHostNetFramework\testhost.net481.arm64.exe
       Microsoft.TestPlatform.Portable\tools\<netcore>\TestHostNetFramework\testhost.net481.arm64.exe.config
       Microsoft.TestPlatform.Portable\tools\<netcore>\TestHostNetFramework\testhost.net481.exe
       Microsoft.TestPlatform.Portable\tools\<netcore>\TestHostNetFramework\testhost.net481.exe.config
       Microsoft.TestPlatform.Portable\tools\<netcore>\TestHostNetFramework\testhost.net481.x86.exe
       Microsoft.TestPlatform.Portable\tools\<netcore>\TestHostNetFramework\testhost.net481.x86.exe.config
    Old: 591 (592 in verify-nupkg, because of checksum file)
    New: 603 (604 in verify-nupkg, because of checksum file)
    You are skipping tfms.
```

We are adding 12 files 6 for each copy of testhost, because we ship both .net and .net framework vstest.console in this package.

Reply:
.cli above needs to put datacollectors back, the process framework family is chosen by the target dll not by the runner.